src/utils/formatTimeHelpers.test.ts

```diff
@@ -5,7 +5,7 @@ import {
     elapsedTimeInSeconds,
     elapsedTimeInSecondsBetween,
     elapsedTimeInSecondsToFixed
-} from '../../utils/formatTimeHelpers';
+} from './formatTimeHelpers';
```

Comment:
should probably be renamed to dateHelpers, since it does a lot more than formating time

Reply:
maybe it's breaking change, and it should just be deprecated. Instead export it as EchoUtils.Date in index.ts

Reply:
Yeah, I would agree with the renaming.

On the dot notation: as I mentioned before I have mixed feelings with that. We saw that with this setup the jsdoc isn't visible, also we don't know if tree shaking would work, since one big object is exported.

I'm experimenting with a package setup where the users can import things from subfolders, like:
```ts
import { formatDate } from '@equinor/echo-utils/dateHelpers';
import { formatString} from '@equinor/echo-utils/stringHelpers';
```

I think this should be the way, but I wasn't to make the build work just yet.

Reply:
Renamed file to dateHelpers

Reply:
Regarding the tree shaking: Most of these util functions are so small, that the size won't make any practical difference.

The big advantage with the dot notation is that I don't need to read the whole documentation every time I want to do something. If I have a string, I can just write stringHelper. and easily see all available functions. This pro outweighs any cons in my opinion :)

We can also export in both format, both as dot notation and as a function if we want to. 

jsdoc, it should be possible to find a solution for this, we just have to investigate it a bit.